zookeeper-server/src/test/java/org/apache/zookeeper/test/ReconfigTest.java

```diff
@@ -1152,4 +1171,74 @@ private void assertRemotePeerMXBeanAttributes(QuorumServer qs, String beanName)
             getAddrPortFromBean(beanName, "QuorumAddress"));
     }
 
+
+    /*
+     * A helper class to parse / compare server address config lines.
+     * Example: server.1=127.0.0.1:11228:11231|127.0.0.1:11230:11229:participant;0.0.0.0:11227
+     */
+    private static class ServerConfigLine {
```

Comment:
I know about this place in `QuorumPeer`: https://github.com/apache/zookeeper/blob/2dcb5e799ec02a2c6a6c7bad80c47169dc095271/zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/QuorumPeer.java#L240

In my case I want to compare complete server config lines during the test for verification (without caring the syntactical correctness or order of the actual addresses), so what I need is a bit different.

It might be nice to create a re-usable helper class for this, but also I don't mind if this logic gets duplicated in the test code. 

Beside: even nicer would be to use some more extendable config format, like JSON. (but it is out of the scope here)

Reply:
Ok, leave it as it is. THanks.